core/CMakeLists.txt

```diff
@@ -171,6 +171,11 @@ foreach(target IN LISTS target_libraries)
             target_include_directories(${target} PUBLIC $<BUILD_INTERFACE:${dir}>)
         endforeach()
     endforeach()
+    if(TF_PSA_CRYPTO_TEST_DRIVER)
+        target_include_directories(${target}
+            PRIVATE ${TF_PSA_CRYPTO_FRAMEWORK_DIR}/tests/include
+                    ${TF_PSA_CRYPTO_DIR}/tests/include/test)
```

Comment:
Why are we putting `tests/include/test` and not `tests/include` on the include path?